### DISCUSSION TITLE
Falha de segurança

### DISCUSSION THREAD

**Discussion by bbertolani** [frontendbr/forum#1681] [Perguntas e Respostas]
Pessoal, não tenho um forum especifico pra isso então venho perguntar a vocês.
No fim de dezembro fiz uma compra em um e-commerce relativamente grande e encontrei um falha que expoe os dados dos clientes entre eles eu!
Prontamente eu entrei em contato via email, que não foi respondido, via twitter, que tb não foi respodindo e por ultimo encontrei o CEO e o CTO da empresa no Linkedin e também fui ignorado.
Eu queria que isso fosse resolvido o mais breve possivel, bom foi o que eu pensei no começo quando descubri a falha, porém não me restam alternativas! Alguém tem uma boa ideia do que posso fazer nessa situação?

**Comment by eduardosoareszup** [comment]
Você pode procurar algum veículo de comunicação de sua confiança para divulgar isso (Claro, alertando para eles avisarem a empresa antes de publicar a notícia e publicar somente quando o problema for resolvido), tenho certeza que rapidamente eles irão dar um jeito de arrumar. Vi um caso similar com um colega e esse foi o recurso que ele encontrou.

**Comment by ninetails** [comment]
O ideal é não ir publicamente pois podem achar que suas ações seriam para denegrir a imagem da empresa e daí podem acionar o jurídico contra a sua pessoa.

Infelizmente creio que o melhor seria encontrar outros devs dessa empresa já que nenhum `C-level` se pronunciou. Ou ver ações cabíveis legalmente, já que vc é um cliente dessa empresa e se sente lesado pela falta de segurança da mesma em relação aos seus dados (que este também pode dar em nada).

Último caso, se vc se sentir lesado a ponto de não conseguir ignorar que essa empresa possua tal falha, procure outros profissionais de segurança. Pelo menos se algo for a público, não associariam ao teu nome (penso eu que se vc postou aqui, não iria querer que ninguém te procure com advogados e etc).
